src/components/element-editor/basic-details/basic-details.styles.tsx

```diff
@@ -1,6 +1,11 @@
+import { Color } from "@lifesg/react-design-system/color";
 import { Accordion } from "@lifesg/react-design-system/accordion";
 import styled from "styled-components";
 
+interface IProps {
+    $displayAlert: boolean;
```

Comment:
since the component is not rendering the alert, can rename it to be more specific to what it is doing
```suggestion
    $hideTopBorder: boolean;
```